mcp_server/server.py

```
@@ -0,0 +1,345 @@
+# ── Endpoints ────────────────────────────────────────────────────────────────
+
+@app.get("/health")
+async def health() -> dict[str, str]:
+    return {"status": "ok", "service": "mcp-server"}
+
+
+@app.post("/mcp")
+async def mcp_dispatch(request: Request) -> JSONResponse:
+    try:
+        body = await request.json()
+    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)